src/ClimaLSM.jl

```diff
@@ -157,7 +158,9 @@ function initialize(model::AbstractModel)
     return Y, p, coords
 end
 
-initialize(model::NotIncluded{FT}) where {FT} = (;model.model_name => FT[]), (;model.model_name => FT[]), (;model.model_name => FT[])
+initialize(model::NotIncluded{FT}) where {FT} = (; model.model_name => FT[]),
```

Comment:
let's change this definition to `function initialize(...)` since it is on multiple lines

Reply:
the last PR was able to remove NotIncluded, but good point! Will keep an eye out for other multi-line functions defined without `function`.

Reply:
Ah right, good point!